book/scaling.md

```diff
@@ -0,0 +1,45 @@
+# Making Your Yii2 App Scalable
```

Comment:
Sounds too general. "Scalable" may include something like database sharding as well. Do you have any idea for a more narrow name?